src/services/BookService.tsx

```diff
@@ -1,22 +1,39 @@
 import axios, { AxiosResponse } from 'axios'
 
 import TokenService from './TokenService'
-import BookRequest from '../model/BookRequest'
 import BookResponse from '../model/BookResponse'
+import Where from '../model/Where'
+import { BookDetailsRequest } from '../model/BookRequest'
+
 
 const token = TokenService.getAccesToken()
 
+interface GetBooksProps {
+  PageNumber: number,
+  PageLength: number,
+  Where: Where[],
+}
+
 
-const urlParams = (bookRequest: BookRequest) => {
-  return `?PageNumber=${bookRequest.PageNumber.toString()}&PageLength=${bookRequest.PageLength.toString()}`
+const urlParams = ( getBookRequest : GetBooksProps) => {
+  let result = '?'
+  result += `PageNumber=${getBookRequest.PageNumber.toString()}`
+  result += `&PageLength=${getBookRequest.PageLength.toString()}`
+  getBookRequest.Where?.forEach((where) => {
+    if(where.Value !== '' && where.Value != null) {
+      result += `&where=${JSON.stringify(where)}`
+    }
+  })
```

Comment:
can we use axios for creating params and not creating it manually?

Reply:
Yes. I think that i must forward params{PageNumer: ...}. Can I implement this when i finish rent and return task?